frontend/public/components/modals/cluster-update-modal.tsx

```diff
@@ -60,12 +61,13 @@ class ClusterUpdateModal extends PromiseComponent {
           <p>{currentVersion}</p>
         </div>
         <div className="form-group">
-          <label htmlFor="version_dropdown">New Version</label>
+          <label htmlFor="version_dropdown">Select new version</label>
```

Comment:
I believe labels should be headline case per Patternfly. (It's what we use elsewhere in console.)

Reply:
We all have PF4 on the brain. I'll update these.